gnosis/safe/tests/test_signatures.py

```diff
@@ -11,6 +11,9 @@
 
 
 class TestSafeSignature(SafeTestCaseMixin, TestCase):
+    EIP1271_MAGIC_VALUE = "20c13b0b"
+    UPDATED_MAGIC_VALUE = "1626ba7e"
```

Comment:
nitpick: this is still related with EIP1271. I would call it `EIP1271_MAGIC_VALUE_UPDATED` which I think better represents an update on original value.

Reply:
Makes sense, https://github.com/safe-global/safe-eth-py/pull/422/commits/04fdade907f04822b203b1dac9d7e78696c5aed1